src/infra/dev/api/oi.tpl.json

```diff
@@ -289,29 +289,634 @@
+            "required": true,
+            "description": "The Access-Control-Request-Headers request header is used when issuing a preflight request to let the server know which HTTP headers will be used when the actual request is made.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "Access-Control-Request-Method",
+            "in": "header",
+            "required": true,
+            "description": "The Access-Control-Request-Method request header is used when issuing a preflight request to let the server know which HTTP method will be used when the actual request is made.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "Origin",
+            "in": "header",
+            "required": true,
+            "description": "The Origin request header indicates where a fetch originates from.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content, applicable to OPTIONS request with preflight configuration",
+            "headers": {
+              "Access-Control-Allow-Methods": {
+                "description": "Allowable Methods, POST",
+                "schema": {
+                  "type": "string",
+                  "default": "POST"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "headers": {
+              "Pragma": {
+                "description": "Pragma browser directive",
+                "schema": {
+                  "type": "string",
+                  "default": "no-cache"
+                }
+              },
+              "Cache-Control": {
+                "description": "Cache-Control browser directive",
+                "schema": {
+                  "type": "string",
+                  "default": "no-store"
+                }
+              }
+            }
+          }
+        },
+        "summary": "OpenID Connect Dynamic Registration Configuration endpoint"
+      },
       "post" : {
+        "tags": ["Authorization Server APIs"],
+        "description": "This endpoint follows the specification defined at http://openid.net/specs/openid-connect-registration-1_0.html. It enables a client to dynamically register itself as an OAuth 2.0 client.",
+        "operationId": "Post_oidc_register",
         "x-amazon-apigateway-integration" : {
+          "passthroughBehavior" : "when_no_match",
+          "contentHandling" : "CONVERT_TO_TEXT",
+          "type" : "aws_proxy",
           "httpMethod" : "POST",
           "uri" : "arn:aws:apigateway:${aws_region}:lambda:path/2015-03-31/functions/${client_registration_lambda_arn}/invocations",
-          "responses" : {},
+          "requestParameters": {
+            "integration.request.header.accept": "'*/*'"
```

Comment:
Should the cors header reach the backend (lambda ) too?

Reply:
I guess they could be handled by the apigw itself